code-studio/src/css/plc/teacher_dashboard.scss

```diff
@@ -24,3 +24,47 @@
   }
 }
 
+.course_unit_sections {
+  display: flex;
```

Comment:
IE 11 - 
![image](https://cloud.githubusercontent.com/assets/14246419/15061066/cfe40214-12e5-11e6-8737-61623dc79852.png)

IE 9 doesn't support it but doesn't look terrible
![image](https://cloud.githubusercontent.com/assets/14246419/15061088/fdf9e952-12e5-11e6-94c4-b2eb6b344154.png)